ci/ci

```diff
@@ -106,11 +114,12 @@ def log_warning(str_to_print):
 def get_services_list():
     services = []
     docker_container_process = subprocess.run(
-        ["docker-compose", "-f", "docker-compose.json", "ps"], stdout=subprocess.PIPE)
-    docker_container = str(docker_container_process.stdout)
-    for val in docker_container.split('\\n'):
-        if val.find('Up') != -1:
-            services.append(val.split(' ')[0])
+        ["docker-compose", "-f", "docker-compose.json",
+         "ps", "--filter", "status=running", "--services"], stdout=subprocess.PIPE)
+    docker_container = str((docker_container_process.stdout).decode('utf-8'))
+    for val in docker_container.split('\n'):
```

Comment:
instead of this for loop you can do like `return [x for x in docker_container.splitlines() if len(x) > 0]` also get rid of services variable